### ISSUE TITLE
Dealing with multiple classifications from one user of a single subject 

### ISSUE THREAD

**Issue by mkosmala** [zooniverse/Panoptes#1741]
@camallen When did the "already_seen" indicator get added to the classifications? I've got a pairs of classifications for the same user and subject from last July and can't see any way to distinguish the two classifications.

Interestingly, the subject has 5 classifications (the threshold for retirement) and is not retired -- presumably because of the duplicate classification? So there must be some awareness that only 4 unique people have seen the subject...

[duplicate_classification_example.txt](https://github.com/zooniverse/Panoptes/files/184362/duplicate_classification_example.txt)

**Comment by camallen** [comment]
@mkosmala looks like we added the ability for the clients to specify this in https://github.com/zooniverse/Panoptes/pull/1215 (20 May 2015) but the client wasn't submitting this data properly https://github.com/zooniverse/Panoptes-Front-End/issues/368#issuecomment-120913401 (13 Jul 2015). 

We then pushed this reponsibility to the backend in https://github.com/zooniverse/Panoptes/pull/1215 (29 Jul 2015) but note this won't be retrospective from that date (or when it was deployed around then).

I'd take the first classification by time as that arrived at the api before the second (15mins before).

Re the retirement limit, most likely due to the duplicate seems the [code at the time](https://github.com/camallen/Panoptes/blob/2ca241acf73afb68fff889f705059d327c7f6810/lib/classification_lifecycle.rb) checked if it was already seen before incrementing the retirment counter.

``` ruby
  def should_count_towards_retirement?
    if !classification.complete? || classification.seen_before?
      false
    else
      classification.anonymous? || subjects_are_unseen_by_user?
    end
  end

  def subjects_are_unseen_by_user?
    return @unseen if @unseen
    @unseen = !UserSeenSubject.find_by!(user: user, workflow: workflow)
      .try(:subjects_seen?, subject_ids)
  rescue ActiveRecord::RecordNotFound
    @unseen = true
  end
```
